Library/Homebrew/cmd/cleanup.rb

```diff
@@ -22,6 +22,8 @@ def cleanup_args
                           "If you want to remove everything, use `--prune=all`."
       switch "-n", "--dry-run",
              description: "Show what would be removed, but do not actually remove anything."
+      switch "-q", "--quiet",
+             description: "Make some output more quiet."
```

Comment:
This section can be omitted if the flag description isn't being customized, e.g. "Suppress listing skipped paths."

Reply:
Sorry which section should I omit? Can you point to another example in the code that I can adapt or use ` ```suggestion ` to give an example?

Reply:
Oh I understand now, nevermind